pxd.c

```diff
@@ -1230,14 +1230,6 @@ static int pxd_init_disk(struct pxd_device *pxd_dev)
 	q->queuedata = pxd_dev;
 	pxd_dev->disk = disk;
 
-#if defined __PX_BLKMQ__ && !defined __PXD_BIO_MAKEREQ__
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(6,14,0) || defined(__EL9_STREAM__)
-	*blk_mq_queue_flag = blk_mq_freeze_queue(q);
-#else
-	blk_mq_freeze_queue(q);
-#endif
-#endif
```

Comment:
if the PR is for "Centos Stream 10 compatibility qualification" then the changes in this file is unrelated.
Also, how is this compiling... the variable blk_mq_queue_flag is already removed?